support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -171,9 +171,8 @@ export function Checkout({ geoId, appConfig, abParticipations }: Props) {
 			billingPeriod,
 		);
 
-		const discountedPrice = promotion?.discountedPrice
-			? promotion.discountedPrice
-			: undefined;
+		const discountedPrice =
+			promotion !== undefined ? promotion.discountedPrice : undefined;
```

Comment:
Would `const discountedPrice = promotion?.discountedPrice ?? undefined` be clearer here? I think that's the same.